nncf/scopes.py

```diff
@@ -81,26 +81,25 @@ def convert_ignored_scope_to_list(ignored_scope: Optional[IgnoredScope]) -> List
     results = []
     if ignored_scope is None:
         return results
-    if ignored_scope.names is not None:
-        results.extend(ignored_scope.names)
-    if ignored_scope.patterns is not None:
-        for p in ignored_scope.patterns:
-            results.append('{re}' + p)
-    if ignored_scope.types is not None:
+    results.extend(ignored_scope.names)
+    for p in ignored_scope.patterns:
+        results.append('{re}' + p)
+    if ignored_scope.types:
         raise RuntimeError('Legacy ignored scope format does not support '
                            'operation types')
     return results
 
 
 def get_ignored_node_names_from_ignored_scope(ignored_scope: IgnoredScope,
-                                              nncf_graph: NNCFGraph) -> List[str]:
+                                              nncf_graph: NNCFGraph, strict: bool = True) -> List[str]:
```

Comment:
Why do we need `strict` parameter?

Reply:
To not raise RuntimeError if we don't need it.
Added dosctring